src/pages/Checkout/CheckoutPage.vue

```diff
@@ -115,7 +115,7 @@
 									@updating-totals="setUpdatingTotals" />
 
 								<kiva-credit-payment
-									v-else
+									v-if="showKivaCreditButton"
```

Comment:
With the addition of the v-if for Braintree above we broke the simple v-else for this. Now we have an explicit check tied to the need for credit, intentionally independent from Paypal and Braintree which will both be active when credit is needed.

Reply:
✔️